### ISSUE TITLE
Pi4 not connecting and sync

### ISSUE THREAD

**Issue by Stepyon** [Chia-Network/chia-blockchain#9497]
What happened?
Pi 4 8gb running latest GUI ubuntu mate 20.04.3 LTS (Focal Fossa).

After reboot can't connect and sync anymore.
Tried to re-install, reboot, didn't help.

Chia version 1.2.11 GUI

See log screenshots attached.
![chia_log](https://user-images.githubusercontent.com/23343477/144946260-2ce0fc98-b3f9-4ce1-bbb6-42e9bede7352.jpg)
![chia_gui](https://user-images.githubusercontent.com/23343477/144946261-f19c571e-3dc2-4c20-8d1f-dafcd5514eb2.jpg)

**Comment by jordanbray** [comment]
To my eyes, this looks like a bug here: https://github.com/Chia-Network/chia-blockchain/blob/main/chia/plotters/madmax.py#L59

`version` must be set to `None` before the `try`.

From a support perspective, it seems like your chia instance _can_ find madmax, but it cannot run it, which is causing problems.  You could frankly delete it by my read of the code, more sensibly check to make sure you can run the executable.  You should be able to find it with a `find ./ | grep madmax-plotter/build/chia_plot`, and try to run the command `the_folder/madmax-plotter/build/chia-plot --version` and see what it puts out.

Once you have confirmed you cannot run it (we have correctly identified the problem, yay!), I would try to rebuild it following the instructions in the `madmax-plotter` folder.

**Comment by Stepyon** [comment]
> To my eyes, this looks like a bug here: https://github.com/Chia-Network/chia-blockchain/blob/main/chia/plotters/madmax.py#L59
> 
> `version` must be set to `None` before the `try`.
> 
> From a support perspective, it seems like your chia instance _can_ find madmax, but it cannot run it, which is causing problems. You could frankly delete it by my read of the code, more sensibly check to make sure you can run the executable. You should be able to find it with a `find ./ | grep madmax-plotter/build/chia_plot`, and try to run the command `the_folder/madmax-plotter/build/chia-plot --version` and see what it puts out.
> 
> Once you have confirmed you cannot run it (we have correctly identified the problem, yay!), I would try to rebuild it following the instructions in the `madmax-plotter` folder.

I didn't compile it from the sources, I used GUI installer. The only `madmax` folder I found is `/lib/chia-blockchain/resources/app.asar.unpacked/daemon/madmax`there are 2 files: `chia_plot` and `chia_plot_k34` I tried to update them with 777 permission but it didn't help and I still see the same error.

**Comment by jordanbray** [comment]
Can you try running that `chia_plot` with `chia_plot --version` and see what it spits out?

**Comment by Stepyon** [comment]
> Can you try running that `chia_plot` with `chia_plot --version` and see what it spits out?
![chia_plot](https://user-images.githubusercontent.com/23343477/145057006-309dcf6f-4327-4813-9a43-8590744e25ff.jpg)

**Comment by loppefaaret** [comment]
try doing `./chia_plot --version` while in that folder

**Comment by jordanbray** [comment]
Are you certain there is no other `chia_plot` executable on the system?  By my read of the code, the software _must_ have found one in a `build` folder

**Comment by Stepyon** [comment]
> Are you certain there is no other `chia_plot` executable on the system? By my read of the code, the software _must_ have found one in a `build` folder

Which `build` folder?
![Screenshot_20211208-033926__01](https://user-images.githubusercontent.com/23343477/145144458-76fc8983-d674-475e-a280-5d31d8cc706b.jpg)

**Comment by Stepyon** [comment]
The madmax issue was not related to sync and connection problem. I installed previous version which has no madmax and it also could not connect, just stuck with progress bars, there was nothing in the log file. So what I did is removed the blockchain database (`.chia/mainnet/db` folder) and it connected to the peers and started to sync. Then I installed the latest version again, copied the database from another machine and now it works. It still throwing the same madmax error and doesn't show madmax in the dropbox but I don't need it to farm on Pi4 anyway.
